staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -146,14 +146,25 @@ type Store struct {
 	// specific cases where storage of the value is not appropriate, since
 	// they cannot be watched.
 	Decorator ObjectFunc
+
```

Comment:
The decorator is no longer the "last" exit hook , should we make it more explicit?

Reply:
Good point, will add to comments

Reply:
re-pushed